src/app/products/service/cart.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class CartService {
+  private cartItemsSubject = new BehaviorSubject<any[]>([]);
+  cartItems$ = this.cartItemsSubject.asObservable();
```

Comment:
pro tip:

servis ti se zove CartService i njegov context je Cart.

samim tim ne moras da imas cart prefix za promenjive.

ja bih to nazvao itemsSubject i items$